whitepapers/business-value/Roadmap.md

```diff
@@ -0,0 +1,19 @@
+## OSS for Business Whitepaper Roadmap! 
+
+More Context for reviewing purposes.\
+Contributors working on the OSS in Business Whitepaper will require a roadmap that will provide additional guidance, establish an estimated timeline and track milestones achieved overtime. This roadmap will be made publicly accessible and linked to the readme.md and Contribution.md files.\
+The roadmap file will entail the following details in a structured format with tag of group leads responsible for specific sections.\
+Suggested format: 
+ <summary><h2> Engineering Driven vs Business Driven in Open Source </h2></summary>
+ 
+- Overview
+- Engineering Driven Projects
```

Comment:
Since the main goal of the guide is to show the value of open source to business managers, and those teams who are not engineering driven, I think this section is not so critical for this project, but more "business-driven projects" / "business use cases"

Reply:
This was me showing an example of what it would look like. The contents and milestones will be determine by the working group in general.